docker-compose.yml

```diff
@@ -1,27 +1,40 @@
-db:
-    image: mongo:3.2
-    command: mongod
-    volumes:
-      - ./data/db:/data/db
-    expose:
-      - "27017"
-    ports:
-    - "27017:27017"
+version: "3"
```

Comment:
Maybe use a different `docker-compose` file with tests services?

Reply:
This is too complex for most to handle:
- you need to switch each time you want to run tests (Most of the time I run my tests while the server running)
- this is not a production compose file
- this means we have to complexify the dev/test documentation
- one more file to the root :/

But it's open to debate. We can provide a second docker-compose file with these services to reduce memory usage for those who ain't running tests, but I think those who wants to test use the official docker image with one of the sample docker-compose.

Reply:
This a just a matter of running `docker-compose -f docker-compose.yml -f docker-compose.tests.yml up` I think. But you're right about "those who wants to test use the official docker image with one of the sample docker-compose", so let's forget this.

Reply:
I changed it anyway, I find it clever to be able to run only whats needed :+1:

Reply:
✌️